client/src/components/SupportTools/SupportTools.tsx

```diff
@@ -235,6 +239,18 @@ const Organization = ({ organizationId }: { organizationId: string }) => {
       }),
     })
 
+  const onClickPermanentlyDeleteAudit = ({ auditName, id }: IElectionBase) => {
+    confirm({
+      title: 'Confirm',
+      description: `Are you sure you want to permanently delete ${auditName}?`,
+      yesButtonLabel: 'Delete',
+      onYesClick: async () => {
+        await deleteElection.mutateAsync({ electionId: id, organizationId })
+        toast.success(`Deleted ${auditName}`)
+      },
+    })
+  }
+
```

Comment:
does this provide some visual feedback while it's deleting, or does the screen freeze until success?

Reply:
there's a spinner on the button and it disables